README.md

```diff
@@ -150,9 +150,10 @@ preLoaders: [
 #### Releasing this repo
 Configure your AWS CLI tools: http://docs.aws.amazon.com/cli/latest/userguide/installing.html.
 
-Run `yarn publish` and enter the new version. This will build and publish the new version, as well as pushing static files to S3.
+Run `yarn release` to cut a new release, this will also generate and commit the CHANGELOG.md automatically.
 
-After publishing, commit the version change and push to master.
+If everything is ok, you can publish the release to github and npm with `yarn publish`, entering the new version you just created.
+This will build, publish and then push static files to S3 for the docs site.
```

Comment:
This is great! 

I'm only concerned about some edge case implications:

* How could you revert a `yarn release` if you change your mind or decided to add some more stuff before `yarn publish`? I imagine running the `yarn release` the second time adds a new entry to the `CHANGELOG.md`? Is there a nice way to squash it then before running `yarn publish`? Thought I should ask before trying it so that I don't mess up the Github `CHANGELOG` :)
* What happens if `yarn publish` fails because on S3 for example? Will we still need to publish a new release then before publishing? (probably not related to this PR, but maybe attempting the S3 upload earlier in the process would prevent these cases)